src/main/java/org/intermine/biovalidator/api/DefaultValidationResult.java

```diff
@@ -69,6 +84,11 @@ public boolean isValid() {
         return isValid;
     }
 
+    @Override
+    public boolean isNotValid() {
+        return !isValid;
```

Comment:
should this return false?

Reply:
isNotValid() is just a helper method so that I don't have to always negate isValid() method call while checking result is valid of in-valid